jcache/embedded/src/main/java/org/infinispan/jcache/embedded/functions/Invoke.java

```diff
@@ -28,13 +29,17 @@
    private final EntryProcessor<K, V, R> processor;
    private final Object[] arguments;
    private final boolean storeByReference;
+   private final DataConversion key;
+   private final DataConversion value;
    private PersistenceMarshaller marshaller;
    private ExpiryPolicy expiryPolicy;
 
-   public Invoke(EntryProcessor<K, V, R> processor, Object[] arguments, boolean storeByReference) {
+   public Invoke(EntryProcessor<K, V, R> processor, Object[] arguments, boolean storeByReference, DataConversion key, DataConversion value) {
```

Comment:
Is this really necessary? I thought JCache doesn't allow the user to override the data conversion.

Also the field/parameter names should include `Conversion`, `key` and `value` is a bit confusing.

Reply:
hmm, I could try to grab from the cache directly during entry processing, let me try

Reply:
Indeed, this is not needed :+1: